templates/semantic-ui/invenio_app_rdm/frontpage.html

```diff
@@ -22,46 +22,41 @@
   {%- endblock intro_section %}
 
   {%- block grid_section %}
-    <div class="ui container overlapping">
+    <div class="ui container rel-mt-2" id="focus-on">
       <div class="ui stackable grid center-tablet-mobile">
-        <div class="column eight wide transparent-background pl-0 pr-0 rel-pb-3">
-          <div class="highlight-background communities-box info-box rel-pb-3">
-            <div class="ui grid middle aligned">
-              <div class="sixteen wide column"><h2 class="ui header">Communities</h2>
-              </div>
-              <div class="sixteen wide mobile ten wide computer sixteen wide tablet column">
-                <p> Browse the catalogue <br/> curated by CDS users and communities</p>
-              </div>
-              <div class="sixteen wide mobile six wide computer sixteen wide tablet column centered">
-                <i class="huge grey users icon mb-20"></i><br/>
-                <a href="/communities" class="ui secondary button">See all communities</a>
-              </div>
-            </div>
-
-          </div>
-        </div>
 
-        <div class="column eight wide inversed-background info-box">
+        <div class="column nine wide ">
           <div class="ui grid relaxed middle aligned">
-            <div class="sixteen wide column">
-              <h2 class="ui header rel-mb-2">Focus on</h2>
+            <div class="three wide column">
+              <h3 class="ui header">Focus on</h3>
             </div>
-            <a href="/search?&f=resource_type:publication" class="eight wide mobile four wide computer eight wide tablet column centered">
-              <i class="huge file alternate icon mb-15"></i><br/>
-              <p>Publications</p>
-            </a>
-            <a href="/search?&f=resource_type:presentation" class="eight wide mobile four wide computer eight wide tablet column centered">
-              <i class="huge group icon mb-15"></i><br/>
-              <p>Presentations</p>
-            </a>
-            <a href="https://videos.cern.ch/" class="eight wide mobile four wide computer eight wide tablet column centered">
-              <i class="huge film icon mb-15"></i><br/>
-              <p>Videos</p>
-            </a>
-            <a href="https://catalogue.library.cern/" class="eight wide mobile four wide computer eight wide tablet column centered">
-              <i class="huge book icon mb-15"></i><br/>
-              <p>Books</p>
-            </a>
+            <button class="ui labeled icon button basic">
+              <i class="file alternate icon"></i>
+              <a href="/search?&f=resource_type:publication">
+                <p>Publications</p>
+              </a>
+            </button>
```

Comment:
```suggestion
            <a href="/search?&f=resource_type:publication" class="ui labeled icon button basic">
              <i class="file alternate icon" aria-hidden="true"></i>
                Publications
            </a>
```
Same for all the same buttons below.

Reply:
is there any reason to change it ?

Reply:
For a11y and screereaders!

Reply:
I think we should document it then, otherwise we normally follow semantic ui docs